drivers/console/Kconfig.ws

```diff
@@ -0,0 +1,112 @@
+# Kconfig - console driver configuration options
+
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig WS_CONSOLE
+	bool "Enable websocket console service"
+	default n
+	select NETWORKING
+	select NET_TCP
+	select HTTP_PARSER
+	select NET_APP_SERVER
+	select WS
+	help
+	This option enables console over a websocket link. Currently,
+	it is basically just a redirection of the Zephyr console through
+	websocket. It nicely works along with another console driver (like
+	uart), twist being that it will take over the output only if a
+	successful connection to its HTTP service is done.
+
+if WS_CONSOLE
+
+config WS_CONSOLE_LINE_BUF_SIZE
+	int "WS console line buffer size"
+	default 128
+	help
+	This option can be used to modify the size of the buffer storing
+	console output line, prior to sending it through the network.
+	Of course an output line can be longer than such size, it just
+	means sending it will start as soon as it reaches this size.
+	It really depends on what type of output is expected.
+	If there is a lot of short lines, then lower this value. If there
+	are longer lines, then raise this value.
+
+config WS_CONSOLE_LINE_BUF_NUMBERS
+	int "WS console line buffers"
+	default 4
+	help
+	This option can be used to modify the amount of line buffers the
+	driver can use. It really depends on how much output is meant to be
+	sent, depending on the system load etc. You can play on both
+	WS_CONSOLE_LINE_BUF_SIZE and this current option to get the
+	best possible buffer settings you need.
+
+config WS_CONSOLE_SEND_TIMEOUT
+	int "WS console line send timeout"
+	default 100
+	help
+	This option can be used to modify the duration of the timer that kick
```

Comment:
what's the time unit here? what happens with the timer "kicks in"?  What's the significance of the "line feed" (or is is a newline? Needs a rewrite, maybe:

      This option can be used to set the time limit (in milliseconds) before
      a non-empty line buffer without a linefeed character is sent.